ttnn/cpp/ttnn/operations/moreh/moreh_adamw/device/kernels/moreh_adamw.cpp

```diff
@@ -129,33 +134,20 @@ void MAIN {
         // denom = sqrt(max_exp_avg_sq) / sqrt(bias_correction2) + eps;
         // denom = sqrt(exp_avg_sq) / sqrt(bias_correction2) + eps;
         // bias_correction2 = 1 - pow(beta2, step);
-        // cb_tmp1 = pow(beta2, step);
-        tile_regs_acquire();
-        cb_reserve_back(cb_tmp1, onetile);
-        copy_tile_init_with_dt(cb_scalar_args);
-        copy_tile(cb_scalar_args, beta2_tile, dst0);
-        power_tile_init();
-        power_tile(dst0, step);
-        tile_regs_commit();
-
-        tile_regs_wait();
-        pack_tile_with_dt(dst0, cb_tmp1);
-        cb_push_back(cb_tmp1, onetile);
-        tile_regs_release();
+        // cb_beta2_exponent = pow(beta2, step); Calculated from host
 
         // cb_tmp1 = 1 / (1 - cb_tmp1);
         tile_regs_acquire();
-        cb_wait_front(cb_tmp1, onetile);
+        // cb_wait_front(cb_tmp1, onetile);
```

Comment:
please remove comment here

Reply:
Resolved in latest commit [b4b7780](https://github.com/tenstorrent/tt-metal/pull/14927/commits/b4b7780ce79a35d25b5313484997200db5cc25e3)